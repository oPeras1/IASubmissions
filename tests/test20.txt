1	2	2	2	3	3	3	4	4	4	5	5	5	5	5	6	6	6	7	8
1	2	9	9	9	3	10	4	4	11	5	12	12	12	12	12	6	7	7	8
1	13	9	9	9	9	10	10	11	11	5	5	5	5	5	7	7	7	8	8
1	13	13	14	9	9	10	10	11	11	15	15	5	5	7	7	5	8	8	16
1	13	14	14	9	9	17	10	10	11	15	15	5	5	5	5	5	16	16	16
1	13	14	14	9	17	17	10	18	11	11	15	19	19	19	5	16	16	20	16
1	1	1	21	17	17	10	10	18	11	22	22	19	23	23	23	23	20	20	20
21	21	21	21	17	24	10	10	18	11	22	22	25	23	25	23	23	23	23	20
26	26	21	17	17	24	10	10	18	11	22	25	25	25	25	27	27	27	23	23
26	26	26	28	24	24	28	10	28	28	22	22	22	29	29	27	30	23	23	23
31	26	26	28	28	28	28	28	28	28	28	22	22	29	29	29	30	30	23	32
31	31	33	33	33	28	28	34	34	28	28	35	35	35	29	29	30	30	32	32
31	31	33	36	36	28	37	37	34	28	28	35	29	29	29	38	38	30	32	32
39	40	40	40	36	36	37	37	34	28	41	41	41	29	38	38	38	42	32	42
39	39	40	40	36	36	37	43	44	45	45	41	38	38	38	46	42	42	42	42
39	39	39	39	36	36	37	43	44	44	45	47	38	38	38	46	46	46	46	46
48	48	48	39	39	37	37	43	43	44	45	47	47	38	49	49	50	50	50	46
39	48	39	39	39	51	51	43	43	45	45	47	47	49	49	49	50	50	52	46
39	39	39	53	53	51	51	51	43	45	45	47	47	54	54	49	50	50	52	52
39	39	53	53	53	43	43	43	43	43	45	47	54	54	49	49	52	52	52	52