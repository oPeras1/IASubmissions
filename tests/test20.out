1	L	L	L	T	T	T	L	L	L	5	5	5	5	5	T	T	T	7	L
1	L	9	9	9	T	10	L	4	T	5	I	I	I	I	12	T	7	7	L
1	T	9	9	L	L	10	10	T	T	5	5	5	T	5	S	S	7	L	L
1	T	T	L	9	L	S	S	11	T	S	15	T	T	S	S	5	8	8	T
L	T	14	L	9	L	17	S	S	11	S	S	5	T	5	5	5	16	T	T
L	13	L	L	9	S	S	10	I	11	11	S	L	L	L	5	16	16	20	T
L	L	1	21	S	S	10	10	I	11	I	22	L	23	T	T	T	L	L	L
21	T	T	T	17	L	10	10	I	11	I	22	T	23	25	T	23	23	23	L
L	26	T	17	17	L	10	10	I	11	I	T	T	T	25	L	L	L	23	23
L	L	L	28	L	L	28	10	28	28	I	22	22	29	29	L	30	23	23	23
T	26	26	28	28	T	28	28	28	28	28	22	22	29	29	S	L	L	23	T
T	T	L	L	L	T	T	L	L	28	28	L	L	L	S	S	30	L	T	T
T	31	L	36	36	T	37	37	L	28	28	L	29	29	S	38	38	L	32	T
39	T	T	T	36	S	37	37	L	28	T	T	T	29	T	38	38	T	32	42
39	39	T	40	S	S	L	43	S	45	45	T	38	T	T	46	T	T	T	42
39	39	39	L	S	36	L	43	S	S	45	S	38	38	T	46	46	46	L	L
T	T	T	L	39	L	L	43	43	S	I	S	S	38	L	L	50	T	50	L
39	T	39	L	L	51	T	43	L	45	I	47	S	49	49	L	T	T	52	L
39	39	39	T	53	T	T	T	L	45	I	47	47	S	S	L	50	T	52	52
39	39	T	T	T	43	43	43	L	L	I	47	S	S	49	49	I	I	I	I